pkg/models/commandvalue.go

```diff
@@ -429,6 +429,16 @@ func (cv *CommandValue) BinaryValue() ([]byte, error) {
 	return value, nil
 }
 
+// ObjectValue returns the value in object data type, and returns error if the Type is not Object.
+func (cv *CommandValue) ObjectValue() (interface{}, error) {
+	var value float64
```

Comment:
Why using `float64` here and not `interface{}`?
I see that it is a noop for the error, but it makes the code confusing.

Reply:
Sorry, it's redundant, removed.